receiver/opencensusreceiver/config.go

```diff
@@ -32,6 +32,8 @@ type Config struct {
 	// TLSCredentials is a (cert_file, key_file) configuration.
 	TLSCredentials *tlsCredentials `mapstructure:"tls-credentials,omitempty"`
 
+	CorsOrigins []string `mapstructure:"corsOrigins"`
```

Comment:
Please add document and update readme about this config.

Reply:
👍

Reply:
So there's actually documentation already there - I just removed the TODO saying it didn't work.

Reply:
GoDoc is still needed, please add.